### PR TITLE
Update CDK doc links, install instructions

### PR DESCRIPTION
**Pull request by rkratky**
Please merge updates of CDK 2.1 doc links (the Docs and APIs section) and installation instructions (the Get Started section).

### CODE REVIEW

**Comment by RubyGemTSL** [comment]
this is ok to test

**Comment by RubyGemTSL** [comment]
tests will fail due to failing tests being pushed over the weekend. I will have a PR merged soon to fix this.

**Comment by LightGuard** [comment]
retest this please

**Comment by rkratky** [comment]
I accidentally added commits from PR #1228 to this PR. Feel free to remove the last three commits from this PR.

**Comment by rkratky** [comment]
Is there any way I can help to speed this up? It's rather unfortunate that old docs are still referenced from the Dev Portal.

**Comment by paulrobinson** [comment]
Apologies, it looks like this has fallen through the cracks. There's a lot
of stuff going on right now, with it being Summit week.

I'll take a look once I next get access to my laptop. Please remind me if
this gets forgotten again.

On 29 Jun 2016, at 10:17, rkratky <notifications@github.com> wrote:

Is there any way I can help to speed this up? It's rather unfortunate that
old docs are still referenced from the Dev Portal.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/redhat-developer/developers.redhat.com/pull/1225#issuecomment-229425266>,
or mute the thread
<https://github.com/notifications/unsubscribe/AADJ4fi57P8oYy81oY6Rhxjj1prYXNumks5qQqifgaJpZM4I_S_a>
.

**Comment by LightGuard** [comment]
@paulrobinson Pretty sure this was forgotten again... I don't know if it's good to merge or not.

**Comment by rkratky** [comment]
Bumping this again. Are there any problems with the PR?

File: products/cdk/get-started-cdk2-ruby.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ include::products/cdk/_includes/get-started-cdk2-step2b.adoc[]
 
 In this step, you will create a tiny Hello World container that uses {tthw-langshort} as a web server. Once created, the container can be run on other systems that have `docker` installed.  You will need to create several files in an empty directory using your favorite editor, including a `Dockerfile` that describes how to build the container image.
 
-Note: you can edit files on your host system that can be synchronized to your vagrant box with `vagrant rsync`. For more information see _Vagrant synchronizad folders_ in the _CDK Installation Guide_. 
+Note: When you use SSHFS, your home directory from the host system is automatically synchronized to your Vagrant box. For more information, see https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.1/getting-started-guide/#synced-folders[Vagrant Synchronizad Folders] in the _CDK Getting Started Guide_.
```
Comment (rctrh) [inline products/cdk/get-started-cdk2-ruby.adoc]: FYI, There are 4 files here with the same wording covering Node, PHP, Python, and Ruby to keep in sync.  This commit hits only two. 

With this edit SSHFS seems a little out of context.  Suggest something like:

You can edit files on your host system and have them automatically synchronized to your Vagrant box using SSHFS. See https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.1/getting-started-guide/#synced-folders[Vagrant Synchronizad Folders] in the _CDK Getting Started Guide_.

**Comment by rctrh** [comment]
Did the build fail or has it been already been cleaned up?   Can an admin please trigger a new build.

Looking at the commits, without checking the preview:

CDK commits
* The first two commits f0b2219 and fc2ff76 look good to me.
* I added a comment on 94f2113, 1) there are 4 language TTHWs that should be in sync. 2) suggested a wording change.

What's being done with the DTS commits are they going to get merged with this PR or reverted?   I spot checked the diffs, they look fine.

**Comment by paulrobinson** [comment]
@rkratky I think the problem is that we need someone to do a technical review of the content. Someone, ideally not the PR creator, needs to read over the changes made (via the preview build) and check the content ins correct and that the links do what is expected. The engineering team focuses on checking that the code is sound and that it doesn't break anything. We are not domain experts so we can't check the content. 

We merged a bunch of things pre-summit that didn't get a thorough content review. This resulted in *many* broken things on the site and *many* JIRAs and emails to me pointing out all the problems. We can't let this happen again. 

@rkratky Can you suggest a person to do the content review? You might need multiple people, as I notice the PR covers many products.

**Comment by paulrobinson** [comment]
@rkratky FYI: I'm on PTO this week, so please try and resolve this with the rest of my team.

**Comment by rkratky** [comment]
@rctrh, I modified the SSHFS wording as per your suggestion and added the same note to the other two TTHW files (which I overlooked before -- thanks for pointing that out).

@paulrobinson, I'll try to find people to review the changes. The PR covered two products because I accidentally pushed the changes for the second product to the same branch as the first ones. I've now reverted those commits to keep it organized. There's a separate PR (#1228) for the changes to the second product.

File: products/cdk/get-started-cdk2-nodejs.adoc [hunk 2]
```diff
@@ -63,7 +63,7 @@ include::products/cdk/_includes/get-started-cdk2-step2b.adoc[]
 
 In this step, you will create a tiny Hello World container that uses {tthw-langshort} as a web server. Once created, the container can be run on other systems that have `docker` installed.  You will need to create several files in an empty directory using your favorite editor, including a `Dockerfile` that describes how to build the container image.
 
-Note: you can edit files on your host system that can be synchronized to your vagrant box with `vagrant rsync`. For more information see _Vagrant synchronizad folders_ in the _Container Development Kit Installation Guide_. 
+Note: You can edit files on your host system and have them automatically synchronized to your Vagrant box using SSHFS. See https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.1/getting-started-guide/#synced-folders[Vagrant Synchronizad Folders] in the _CDK Getting Started Guide_.
```
Comment (pmuir) [inline products/cdk/get-started-cdk2-nodejs.adoc]: Just wonder if we want to make this technology-agnostic. IOW the team are investigating other methods of syncing folders as well that may work better - so remove "using SSHFS".

File: products/cdk/get-started.adoc [hunk 3]
```diff
@@ -59,25 +59,25 @@ First, download and install a virtualization platform:
...
+*NOTE:* Hyper-V support is only available on certain versions of Windows, such as Windows 10 Pro. To enable Hyper-V, right click on the _Start Menu_, select _Programs and Features, click on _Turn Windows features on or off_, make sure the checkbox next to _Hyper-V_ is selected, and click _OK_. Windows may need to restart to install Hyper-V - follow any prompts.
 
 ### Install Vagrant
 
 Next, install Vagrant. All of the Container Development Environment configuration and managment on your development system will be handled for you by Vagrant.
 
 [.callout-light]
-*NOTE:* We recommend you use Vagrant version 1.7.4, or a version *later than 1.8.1* on Microsoft Windows and Apple Mac OS X. Vagrant releases 1.8.0 and 1.8.1 should be avoided due to some bugs that impact proper operation of the kit.
+*NOTE:* Use Vagrant version 1.7.4 with VirtualBox on Microsoft Windows. Use version 1.8.1 and higher with Hyper-V. On Red Hat Enterprise Linux, use Vagrant 1.7.4.
```
Comment (pmuir) [inline products/cdk/get-started.adoc]: BTW @LalatenduMohanty hferentschik@ I had problems with latest Windows 10 updates and vagraqnt 1.7.4 (and I had to update virtual box). I'm running Windows Insider (so I get previews) but I suspect we need to be on top of this for when they update windows...

File: products/cdk/get-started.adoc [hunk 4]
```diff
@@ -90,7 +90,7 @@ To provide `ssh` and `rsync` on Microsoft Windows, install http://cygwin.com/ins
 * http://cygwin.com/setup-x86_64.exe[setup-x86_64.exe, window='_blank'] for 64-bit versions of Windows.
 
 
-### Download the Container Runtime Environment
+### Download the Container Development Environment
```
Comment (pmuir) [inline products/cdk/get-started.adoc]: I thought we went for CRE in the end?

Comment (rkratky) [inline products/cdk/get-started.adoc]: Not AFAIK. See https://docs.google.com/document/d/17hbzJY2eV4dwjKO2p_4mOh3Xqno9LgUobOnTHmp4iMo/edit

File: products/cdk/get-started.adoc (hunk 3 above)
Comment (rkratky) [inline products/cdk/get-started.adoc]: This is based on QE findings for CDK 2.1. I can update this when new testing gets done.

File: products/cdk/get-started-cdk2-nodejs.adoc (hunk 2 above)
Comment (rkratky) [inline products/cdk/get-started-cdk2-nodejs.adoc]: Good idea -- the Get Started on dev.rh.com doesn't need to have the implementation details. I'll push a correction.

**Comment by rkratky** [comment]
Pushed changes based on Pete's suggestions and a couple of other fixes.

**Comment by RickJWagner** [comment]
I've validated Robert's recent changes, they correctly fulfill an outstanding JIRA.  (Please allow this PR).

**Pull request merged**